### PR TITLE
fix: ux component qa

### PR DESCRIPTION
**Pull request by dianneweinthal**
QA updates for the following components

- banner text
- block staff list
- flexible blocks
- impact number carousel
- section dual masonry

### CODE REVIEW

File: src/lib-components/BlockStaffList.vue [hunk 1]
```diff
@@ -76,6 +76,14 @@ export default {
             type: String,
             default: "",
         },
+        alternativeFullName: {
```
Comment (jendiamond) [inline src/lib-components/BlockStaffList.vue:79]: I don't think that we will need alternativeFullName.

Comment (axamei) [reply src/lib-components/BlockStaffList.vue:79]: yes, we should remove the props if they are not used.

**Review by axamei — CHANGES_REQUESTED** [review]
exclude files outside of the src/lib-components directory and remove the new props from blockstafflist.vue

**Review by jendiamond — APPROVED** [review]
🍥

**Pull request merged**